src/ontology/mondo-edit.obo

```diff
@@ -143023,8 +143023,8 @@ intersection_of: MONDO:0015281 ! atrial standstill
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10593 ! SCN5A
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4279 ! GJA5
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-multiple-gene-associations.sparql
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10593 {source="OMIM:108770"} ! SCN5A
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4279 {source="MONDO:mim2gene_medgen", source="OMIM:108770"} ! GJA5
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10593 ! SCN5A
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4279 {source="MONDO:mim2gene_medgen"} ! GJA5
```

Comment:
These should have been ignored, and the OMIM source should have been kept. 
What is the protocol to make sure that these annotations are left alone? 
(these are, I believe, on the "ignore" list - though I do not know what the SOP is regarding annotations which should be left along)

Reply:
agree, these should have been maintained are also in the mondo-omim-genes.robot.tsv file